### DISCUSSION TITLE
Backup/Restore Functionality

### DISCUSSION THREAD

**Discussion by TrikkStar** [sct/overseerr#2658] [Feature Requests]
Apologies if this has already been discussed. But I would love to see an implementation of Backup/Restore functionality similar to how the *arr apps work. Mainly so to simplify importing/exporting an existing instance of Overseerr from one system to another.

**Comment by Cobra-James** [comment]
Logged on specifically just to 1 up this. I can tear down and rebuild the arr stack easily with docker and restoring backups, but the 1 I can't is Overseerr. Would love this functionality.

**Reply by Vandekieft** [reply]
Oh yes Please. I just found out about the lack of this feature after I finally sat down to manually upgrade my TrueCharts apps since the BlueFin break...

**Comment by KevinBoone90** [comment]
Yes please! I was wondering the exact same thing, was copying folders from the config over but can’t seem to find an import during installation…

**Comment by Nusephived** [comment]
Up, this is a must!

**Comment by brando56894** [comment]
Kind of disappointing to see that this has been open for longer than a year with zero movement or comments from the devs.

**Comment by Patatra** [comment]
Would love to see that too!

EDIT : i just have to copy the config folder from the old instance to the new one and connect using my plex account to find everything back. Config folder should be accessible from `/opt/overseer/config` if not using docker (you can find it under `settings` - `about`)

**Comment by scallister** [comment]
After recently having an sqlite database corrupt I have a few thoughts on what I'd love to see in a potential implementation. Leaving here for inspiration.

1. Default to a root level folder, for example `/backups`
  The reason for this is because many people like to use `/config` as a mount point, sometimes for the local filesystem to keep apps running quickly. A subsequent mount to `/config/backups` becomes problematic in this case. Having a clear root folder such as `/backups` makes this an easy mount to use and might negate the need for picking a specific folder for most users.
1. The ability to specify a specific folder in some way
  1. Preferably one of:
    a. Text box in app (harder to implement, easy to use)
    b. Env variable (easiest to implement, painful to use)
1. Some kind of rotation of backups. *arr systems allow for specifying both an interval for backing up (X days) and a retention period of X days. 
1. Bit of a stretch, but some kind of auto restore functionality would be fantastic. Currently when an *arr system dies and moves to a new node, one has to restore from a backup. If Overseerr were able to auto restore from the latest backup if it sees there are backups available that would be great. If a root level default of `/backups` were used this would make auto restore even simpler.

**Comment by FizzGiGG** [comment]
> Apologies if this has already been discussed. But I would love to see an implementation of Backup/Restore functionality similar to how the *arr apps work. Mainly so to simplify importing/exporting an existing instance of Overseerr from one system to another.

Would love an automated backup for this.

**Comment by lt-kraken** [comment]
Bump.

**Comment by digital-grease** [comment]
bump

**Comment by richardavenia** [comment]
Bump 🙏

**Comment by gshpychka** [comment]
@sct it is quite disappointing that the most upvoted Discussion post never got even an acknowledgement from the dev team :/

**Comment by joldnl** [comment]
Yes please!! Im doing this manually now :/
